Release/src/http/common/http_compression.cpp

```diff
@@ -17,18 +17,9 @@
 // it. CPPREST_EXCLUDE_BROTLI is set if we want to explicitly disable Brotli compression support.
 // CPPREST_EXCLUDE_WEBSOCKETS is a flag that now essentially means "no external dependencies". TODO: Rename
 
-#if __APPLE__
-#include "TargetConditionals.h"
-#if defined(TARGET_OS_MAC)
-#if !defined(CPPREST_EXCLUDE_COMPRESSION)
-#define CPPREST_HTTP_COMPRESSION
-#endif // !defined(CPPREST_EXCLUDE_COMPRESSION)
-#endif // defined(TARGET_OS_MAC)
-#elif defined(_WIN32) && (!defined(WINAPI_FAMILY) || WINAPI_PARTITION_DESKTOP)
```

Comment:
This condition always returns true, because if WINAPI_FAMILY is defined, then WINAPI_PARTITION_DESKTOP is just a constant.
Perhaps the author wanted to write WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP), but I do not know why this condition is necessary since UWP applications should also work